tests/unit/components/HdGalleryTiles.spec.js

```diff
@@ -8,12 +8,27 @@ describe('HdGalleryTiles', () => {
   beforeEach(() => {
     wrapper = mount(HdGalleryTiles, {
       propsData: {
-        item: ITEMS,
+        items: ITEMS,
       },
     });
   });
 
   it('renders component', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
+
+  it('emits the proper event when the user clicks on an image', () => {
+    const chosenImageIndex = 2;
+    const image = wrapper.findAll('.gallery-tile').at(chosenImageIndex);
+    image.trigger('click');
+
+    expect(wrapper.emitted('itemClick')[0][0]).toBe(chosenImageIndex);
+  });
```

Comment:
I would add the same test covering this event as well: https://github.com/homeday-de/homeday-blocks/blob/develop/src/components/gallery/HdGalleryTiles.vue#L10

Reply:
Done!

Reply:
I think you forgot to push it 😅

Reply:
FFFUUUUU